samples/nrf9160/modem_shell/src/print/print_shell.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <shell/shell.h>
+
+#include "mosh_print.h"
+
+extern bool mosh_print_timestamp_use;
+
+static int print_help(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret = 1;
```

Comment:
You could remove this line and simply "return 1;" 
Does this function even need to be int-type if it always returns 1 regardless of the outcome?

Reply:
True, will just do `return 1;`
Function prototype is coming from a typedef used in SHELL_CMD macros so it has to return `int`.